Rubberduck.Parsing/Inspections/Abstract/IInspectionResult.cs

```diff
@@ -8,6 +8,9 @@ namespace Rubberduck.Parsing.Inspections.Abstract
     public interface IInspectionResult : IComparable<IInspectionResult>, IComparable
     {
         IEnumerable<IQuickFix> QuickFixes { get; }
+        bool HasQuickFixes { get; }
```

Comment:
The ViewModel was working off `InspectionResultBase`, now `IInspectionResult` - I didn't want to change any functionality, just keep the code compilable... so the property had to get onto the interface.